pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1949,13 +1949,20 @@ class HdRprApiImpl {
         float focalLength;
 
         GfVec2f apertureSize;
+        GfVec2f apertureOffset(0.0f);
         HdRprCamera::Projection projection;
         if (m_hdCamera->GetFocalLength(&focalLength) &&
             m_hdCamera->GetApertureSize(&apertureSize) &&
+            m_hdCamera->GetApertureOffset(&apertureOffset) &&
             m_hdCamera->GetProjection(&projection)) {
             ApplyAspectRatioPolicy(m_viewportSize, aspectRatioPolicy.value, apertureSize);
             sensorWidth = apertureSize[0];
             sensorHeight = apertureSize[1];
+
+            apertureOffset[0] /= apertureSize[0];
+            apertureOffset[1] /= apertureSize[1];
+
+            apertureOffset[1] *= aspectRatio;
```

Comment:
Multiplication is not need here

Reply:
Weird because to match with `Houdini GL` delegate this multiplication is needed. I will check Houdini's UI for other settings that might control this behavior

Reply:
yes, this is weird, because I tested in USD blender plugin and comparing to GL delegate this is not needed

Reply:
There is nothing in Houdini to control this behavior. What's more interesting `Karma` renderer results do not match with `Houdini GL`. So I don't know who is right here.

Reply:
`Cycles` renderer does not use aperture offset for a perspective camera at all. So I think the best would be to match with `Storm`.